apidef/oas/upstream.go

```diff
@@ -672,7 +672,7 @@ func (u *UptimeTests) Fill(uptimeTests apidef.UptimeTests) {
 	for _, v := range uptimeTests.CheckList {
 		check := UptimeTest{
 			CheckURL:            u.fillCheckURL(v.Protocol, v.CheckURL),
-			Timeout:             ReadableDuration(v.Timeout),
+			Timeout:             ReadableDuration(v.Timeout * time.Nanosecond),
```

Comment:
This multiplication strikes me as wrong/redundant. v.Timeout and Readable duration are both time.Duration and should not need time math

Reply:
it is indeed redundant in the sense that the value is just multiplied by *1 but I thought being extra specific inside the code would enhance readability in the future (e.g. I had an issue as I couldn't find any piece of documentation telling me what unit of measure we're using for that field)